context/app/static/js/components/search/Results/ResultsTable.tsx

```diff
@@ -90,6 +90,7 @@ function SortHeaderCell({ field, label }: { field: string; label: string }) {
     <StyledHeaderCell>
       {label}
       <IconButton
+        aria-label="Sort Column"
```

Comment:
I was getting the `Ensure buttons have discernible text` message in the dev env, but not prod, so added this.

Reply:
Good catch! This would've been a minor regression owing to the searchkit migration.